functests/3_performance_status/status.go

```diff
@@ -103,6 +115,43 @@ var _ = Describe("Status testing of performance profile", func() {
 
 			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
 		})
+
+		It("[test_id:40402] Tuned profile status tied to Performance Profile", func() {
+			// During this test we're creating additional synthetic tuned CR by invoking the createrBadTuned function.
+			// This synthetic tuned will look for a tuned profile which doesn't exist.
+			// This tuned CR will be applied on the profiles.tuned.openshift.io CR (there is such profile per node)
+			// which is associate to the node object with the same name.
+			// The connection between the node object and the tuned object is via the MachineConfigLables, worker-cnf in our case.
+			ns := "openshift-cluster-node-tuning-operator"
+			tunedName := "openshift-cause-tuned-failure"
+
+			// Make sure to clean badTuned object even if the It threw an assertion
```

Comment:
assertion ?

Reply:
I thought the error being thrown by the ```Eventually``` blocks called assertions. Read that wrong. will fix